server/app/services/seeding/DatabaseSeedTask.java

```diff
@@ -46,7 +46,7 @@ public final class DatabaseSeedTask {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(DatabaseSeedTask.class);
   private static final int MAX_RETRIES = 10;
```

Comment:
Below we revert the phrasing back to "canonical" after #5213 - I didn't realize earlier that canonical does make sense for the DOB and name questions, which are seeded for all entities on server startup.